src/openPr.js

```diff
@@ -75,19 +103,12 @@ module.exports = async function ({ context, inputs, packageVersion }) {
 
   await run('git', ['push', 'origin', branchName])
 
-  const releaseCommitHash = await run('git', ['rev-parse', 'HEAD'])
+  const draftRelease = await createDraftRelease(inputs, newVersion)
 
-  const { data: draftRelease } = await callApi(
-    {
-      method: 'POST',
-      endpoint: 'release',
-      body: {
-        version: newVersion,
-        target: releaseCommitHash,
-      },
-    },
-    inputs
-  )
+  // Halt action here if release draft could not be created for some reason.
+  if (!draftRelease) {
+    throw new Error('Failed to create draft release. Aborting.')
+  }
```

Comment:
in theory this check should be unnecessary. having it means that we're not sure if the resolved value of callApi may be falsy, which in my opinion would signal a bug elsewhere. if we do want to preserve this check, I'd move it inside the createDraftRelease function though, for consistency, so we don't have to do fancy error handling here

Reply:
I agree it shouldn't be necessary but since fetch will only throw on network failure and not if the response is 4xx or 5xx I thought it was a good idea to add that just to make sure we're not missing anything but I removed it now. Maybe we can improve `callApi` later to handle errors better, since now it checks the reponse status but if it's not 200 we just log a warning telling the user to install the github app (I'm not even sure that makes 100% sense since (at least I think) we could have something different causing a server error, not just that the app is not installed)